docs/source/workshop.rst

```diff
@@ -1,4 +1,106 @@
-Workshop Template
-===================
+Creating a conference
+========================
+
+.. toctree::
+   :maxdepth: 3
+
+   icml
+
+Webfield example::
```

Comment:
It is confusing to see the webfield, may be we can add it as a cell in the python notebook so it is close to the creation of the group